bookwyrm/models/user.py

```diff
@@ -173,6 +174,17 @@ class User(OrderedCollectionPageMixin, AbstractUser):
     property_fields = [("following_link", "following")]
     field_tracker = FieldTracker(fields=["name", "avatar"])
 
+    @property
+    def get_theme(self):
```

Comment:
I noticed in testing that this breaks when you're logged out -- the template resolves to empty string.

I assume that this is because the user object is `None` in that case.

Reply:
Thank you! The user model is definitely not the right place for this function